.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Quick Go unit tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      GO111MODULE: off
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        path: ./src/github.com/${{ github.repository }}
+    - name: Cache go pkg directory
+      id: cache-go-pkg
+      uses: actions/cache@v1
+      with:
+        path: ${{ github.workspace }}/pkg
+        key: ${{ runner.os }}-go-pkg
+    - name: Cache go vendor packages
+      id: cache-go-vendor
+      uses: actions/cache@v1
+      with:
+        path: ${{ github.workspace }}/src/github.com/${{ github.repository }}/vendor
+        key: ${{ runner.os }}-go-vendor
+    # XXX: is this the right version of go?
+    - name: Set up go 1.10
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.10
```

Comment:
I think we should run 1.9 and latest just like we do on travis. 1.9 as it's the minimal version we support right now (iirc for centos 7, it would be 1.10 otherwise). And latest to ensure e.g. debian-sid/arch who move fast always work

Reply:
Sure, I'll change this.

Reply:
Done